commitlint.config.ts

```diff
@@ -186,6 +194,25 @@ module.exports = {
                     ];
                 },
 
+                'proper-issue-refs': ({raw}: {raw:any}) => {
```

Comment:
@realmarv you're using a workaround to a bug here ^ (using raw instead of body); let's benefit from your bug filing by adding a comment explaining that this is a workaround, and linking to the bug

Reply:
Done